src/screens/ChatScreen/SignedChatScreen.tsx

```diff
@@ -138,7 +138,7 @@ const SignedChatScreen = () => {
       {!exitedGroup && (
         <View style={styles.inputContainer}>
           <InputMessageV2
-            onSendButtonClicked={sendChat}
+            onSendButtonClicked={sendChatMutation.mutate}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change also reflects the shift from `sendChat` to `sendChatMutation.mutate`. As mentioned above, make sure that `sendChatMutation.mutate` has proper error handling mechanisms in place to prevent application crashes due to failed chat sending operations.
```diff
-            onSendButtonClicked={sendChat}
+            onSendButtonClicked={sendChatMutation.mutate}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->